README.md

```diff
@@ -139,6 +139,96 @@ command groups. The `NullOrchestrator` disables commands related to managing ser
 orchestrator=NullOrchestrator()
 ```
 
+#### HelmOrchestrator
+
+The project also includes a [helm](https://helm.sh/docs/intro/quickstart/) orchestrator for deploying charts to [kubernetes](https://kubernetes.io/) clusters.
+
+Create a new `resources` directory as follows:
+
+```bash
+drwxrwxr-x  resources/
+.rw-rw-r-- ├──  settings.yml
+drwxrwxr-x └──  templates/
+drwxrwxr-x    ├──  baseline/
+drwxrwxr-x    │  └──  cli/
+drwxrwxr-x    │     └──  helm/
+drwxrwxr-x    │        ├──  values/
+.rw-rw-r--    │        │  └──   values.yml
+drwxrwxr-x    │        ├──  values-files/
+.rw-rw-r--    │        └──   mychart.tgz
+drwxrwxr-x    └──  configurable/
+drwxrwxr-x       └──  cli/
+drwxrwxr-x          └──  home/
+drwxrwxr-x             └──  .kube/
+.rw-------                └──  config  # Overwrite this with a cluster specific config file. ie `~/.kube/config`.
+```
+
+You can then define the orchestrator:
+
+```python
+from appcli.orchestrators import HelmOrchestrator
+orchestrator = HelmOrchestrator(
+    # Name of the project.
+    release_name="myapp",
```

Comment:
Why do we want to set another name here? Can't we just use the app name from the `Configuration` class?

Or even `CliContext.get_project_name` if its appropriate to include the release environment to distinguish things.